provider/scaleway/scaleway.go

```diff
@@ -104,6 +105,14 @@ func NewScalewayProvider(ctx context.Context, domainFilter endpoint.DomainFilter
 	}, nil
 }
 
+func (p *ScalewayProvider) GetProviderSpecific(_ context.Context) (apis.ProviderSpecificConfig, error) {
+	return apis.ProviderSpecificConfig{
+		PrefixTranslation: map[string]string{
+			"external-dns.alpha.kubernetes.io/ibmcloud-": "ibmcloud/",
```

Comment:
```suggestion
			"external-dns.alpha.kubernetes.io/scw-": "scw/",
```

Reply:
Btw, is there really scaleway specific annotations ? I cannot find any in external-dns or scaleway tutorials ?

Reply:
fixed. It looks like https://github.com/kubernetes-sigs/external-dns/pull/3959/files#diff-c2c587e9de1acc152a646c08609ac201aa8f5b78494e4ec069d340ad89b2ceb2L215 . Probably never documented.